code-challenges.js

```diff
@@ -0,0 +1,52 @@
+// This file was created to test the legitimacy of the code created for the first problem of the coding challenge. I initially passed the test, but I couldn't tell if my code was actually shuffling the values in the array. After running the code in the terminal through this file, it became apparent that my code was only removing the first value of the array.
+// After much trial and error through this file, I was able to create a code that satisfied the conditions of the prompt.
```

Comment:
You can also console log inside a function inside your test file.  This still allows you to troubleshoot and check outcomes without creating additional files.